_posts/2016-05-02-darcli.markdown

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Darcli
+date: 2016-05-02 17:00:00
+homepage: https://github.com/gjuniioor/darcli
+download: https://github.com/gjuniioor/darcli/archive/master.zip
+demo: https://gjuniioor.github.io/darcli/
+author: Gildásio Júnior aka gjuniioor
+thumbnail: darcli.png
+license: GPL3
```

Comment:
To keep up with similar formatting, please use `GNU General Public License Version 3` as license.